cdap-ui/app/cdap/components/DataPrep/WorkspaceTabs/index.js

```diff
@@ -75,6 +76,7 @@ export default class WorkspaceTabs extends Component {
 
   componentDidMount() {
     this.calculateMaxTabs();
+    MouseTrap.bind('enter', () => this.handleDeleteWorkspace(this.state.deleteWorkspace.id));
```

Comment:
This will work even if I don't have the delete workspace modal, causing a JS error

Reply:
There is a `.unbind` in `componentWillUnmount`?

Reply:
But this is triggered when the workspace tabs render right? Right now if you go to DataPrep (with a workspace open) and just press Enter, there'll be a JS error saying `Cannot read property 'id' of null`

Reply:
Ah got it. Fixing it.